File: app/src/main/java/com/example/niousha/refugeeapp/Volunteer.java
package com.example.niousha.refugeeapp;

/**
 * Created by niousha on 4/9/18.
 */

public class Volunteer {
    String firstName;
    String lastName;
    String phoneNumber;
    String email;

    public Volunteer(String firstName, String lastName, String phoneNumber, String email) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.phoneNumber = phoneNumber;
        this.email = email;
    }

    public String getFirstName() {
        return firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public String getPhoneNumber() {
        return phoneNumber;
    }

    public String getEmail() {
        return email;
    }
}

File: app/src/main/java/com/example/niousha/refugeeapp/Volunteer.kt
package com.example.niousha.refugeeapp

/**
 * Created by niousha on 4/9/18.
 */

class Volunteer(firstName: String, lastName: String, phoneNumber: String, email: String) {
    var firstName: String
        internal set
    var lastName: String
        internal set
    var phoneNumber: String
        internal set
    var email: String
        internal set

    init {
        this.firstName = firstName
        this.lastName = lastName
        this.phoneNumber = phoneNumber
        this.email = email
    }
}
